README.md

```diff
@@ -6,22 +6,21 @@ A sample application and dataset for getting started with Couchbase 4.5 or later
 ## Installation and Configuration
 The steps below assume you are running a standalone couchbase instance running kv, indexing, fts (in Couchbase 4.5 or later) and query services on the same server where the node application will also be running.
 
- - [1] Install a Couchbase Server, and start it.
- 
- - [2] Install Node.js
+1. Install a Couchbase Server, and start it.
+2. Install Node.js
+3. Make a directory, clone this repo, install dependencies, start the application.  From a terminal:
 
- - [3] Make a directory, clone this repo, install dependencies, start the application.  From a terminal:
-
-    **mkidr ~/try-cb   
+    **mkdir ~/try-cb   
     git clone https://github.com/couchbaselabs/try-cb-nodejs.git ~/try-cb   
-    cd ~/try-cb**   
+    cd ~/try-cb
+    npm install**   
 
- - [4] Start the application.  From a terminal:
+4. Start the application.  From a terminal:
 
-    **npm run start**
+    **node index**
```

Comment:
This can be either `npm start` which uses the "main" entry from packages.json to call into app.js, or you could call `npm app.js` directly. I think the normal way is to use the npm version though.